### DISCUSSION TITLE
CDP manage browser tabs

### DISCUSSION THREAD

**Discussion by Wing1159** [cefsharp/CefSharp#4569] [Q&A] [answered]
How to manage browser tabs through CDP?

I managed browser tabs through the following API, but there was no response,
http://127.0.0.1:{RemoteDebuggingPort}/json/json/new?{url}
http://127.0.0.1:{RemoteDebuggingPort}/json/activate/{targetId}
http://127.0.0.1:{RemoteDebuggingPort}/json/close/{targetId}

**Comment by amaitland** [comment, marked as answer]
Not supported.

**Reply by Wing1159** [reply]
Oh,Thank you
